src/CMakeLists.txt

```diff
@@ -37,7 +37,352 @@ else()
+  datebf.F
+  datelen.f
+  digit.f
+  drfini.f
+  drstpl.f
+  dumpbf.f
+  dxdump.f
+  dxinit.f
+  dxmini.f
+  elemdx.f
+  errwrt.f
+  exitbufr.f
+  fstag.f
+  getabdb.f
+  getbmiss.f
+  getcfmng.f
+  getlens.f
+  getntbe.f
+  gets1loc.f
+  gettagpr.f
+  gettagre.f
+  gettbh.f
+  getvalnb.f
+  getwin.f
+  hold4wlc.f
+  i4dy.f
+  ibfms.f
+  icbfms.f
+  ichkstr.f
+  icmpdx.f
+  icopysb.f
+  idn30.f
+  idxmsg.f
+  ifbget.f
+  ifxy.f
+  igetdate.f
+  igetfxy.f
+  igetmxby.f
+  igetntbi.f
+  igetntbl.f
+  igetprm.f
+  igetrfel.f
+  igetsc.f
+  igettdi.f
+  imrkopr.f
+  inctab.f
+  invcon.f
+  invmrg.f
+  invtag.f
+  invwin.f
+  iok2cpy.f
+  iokoper.f
+  ipkm.f
+  ipks.f
+  ireadmg.f
+  ireadmm.f
+  ireadns.f
+  ireadsb.f
+  ishrdx.f
+  isize.f
+  istdesc.f
+  iupb.f
+  iupbs01.f
+  iupbs3.f
+  iupm.f
+  iupvs01.f
+  jstchr.f
+  jstnum.f
+  lcmgdf.f
+  lmsg.f
+  lstjpb.f
+  makestab.f
+  maxout.f
+  mesgbc.f
+  mesgbf.f
+  minimg.f
+  mrginv.f
+  msgfull.f
+  msgini.f
+  msgupd.f
+  msgwrt.f
+  mtfnam.f
+  mtinfo.f
+  mvb.f
+  nemdefs.f
+  nemock.f
+  nemspecs.f
+  nemtab.f
+  nemtba.f
+  nemtbax.f
+  nemtbb.f
+  nemtbd.f
+  nenubd.f
+  nevn.f
+  newwin.f
+  nmsub.f
+  nmwrd.f
+  numbck.f
+  numtab.f
+  numtbd.f
+  nvnwin.f
+  nwords.f
+  nxtwin.f
+  openbt.f
+  openmb.f
+  openmg.f
+  pad.f
+  padmsg.f
+  parstr.f
+  parusr.f
+  parutg.f
+  pkb.f
+  pkbs1.f
+  pkc.f
+  pkftbv.f
+  pktdd.f
+  posapx.f
+  rcstpl.f
+  rdbfdx.f
+  rdcmps.f
+  rdmemm.f
+  rdmems.f
+  rdmgsb.f
+  rdmsgw.f
+  rdmtbb.f
+  rdmtbd.f
+  rdmtbf.f
+  rdtree.f
+  rdusdx.f
+  readdx.f
+  readerme.f
+  readlc.f
+  readmg.f
+  readmm.f
+  readns.f
+  reads3.f
+  readsb.f
+  rewnbf.f
+  rjust.f
+  rsvfvm.f
+  rtrcpt.f
+  rtrcptb.f
+  seqsdx.f
+  setblock.f
+  setbmiss.f
+  setvalnb.f
+  sntbbe.f
+  sntbde.f
+  sntbfe.f
+  status.f
+  stbfdx.f
+  stdmsg.f
+  stndrd.f
+  stntbi.f
+  stntbia.f
+  strbtm.f
+  strcln.f
+  strcpt.f
+  string.f
+  strnum.f
+  strsuc.f
+  tabent.f
+  tabsub.f
+  trybump.f
+  ufbcnt.f
+  ufbcpy.f
+  ufbcup.f
+  ufbdmp.f
+  ufbevn.f
+  ufbget.f
+  ufbin3.f
+  ufbint.f
+  ufbinx.f
+  ufbmem.f
+  ufbmex.f
+  ufbmms.f
+  ufbmns.f
+  ufbovr.f
+  ufbpos.f
+  ufbqcd.f
+  ufbqcp.f
+  ufbrep.f
+  ufbrms.f
+  ufbrp.f
+  ufbrw.f
+  ufbseq.f
+  ufbsp.f
+  ufbstp.f
+  ufbtab.f
+  ufbtam.f
+  ufdump.f
+  upb.f
+  upbb.f
+  upc.f
+  upds3.f
+  upftbv.f
+  ups.f
+  uptdd.f
+  usrtpl.f
+  valx.f
+  wrcmps.f
+  wrdxtb.f
+  writcp.f
+  writdx.f
+  writlc.f
+  writsa.f
+  writsb.f
+  wrtree.f
+  wtstat.f
+  arallocf.F
+  ardllocf.F
+  closbf.F
+  ireadmt.F
+  irev.F
+  isetprm.F
+  openbf.F
+  pkvs01.F
+  wrdlen.F
+  fortran_open.f90
+  fortran_close.f90
+  bufr_interface.f90)
+
+set(c_src
+  arallocc.c
+  ardllocc.c
+  bort_exit.c
+  ccbfl.c
+  cmpia.c
+  cmpstia1.c
+  cmpstia2.c
+  cobfl.c
+  cpmstabs.c
+  crbmg.c
+  cread.c
+  cwbmg.c
+  dlloctbf.c
+  icvidx.c
+  inittbf.c
+  nummtb.c
+  rbytes.c
+  restd.c
+  sorttbf.c
+  srchtbf.c
+  strtbfe.c
+  stseq.c
+  wrdesc.c)
```

Comment:
I really don't think they each need their own line...

Reply:
You're right, they probably don't.  If you'd prefer I could put a bunch on each line to reduce the overall number of file lines.